sycl/test/function-pointers/fp-as-kernel-arg.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -Xclang -fsycl-allow-func-ptr -std=c++14 -fsycl %s -o %t.out -lOpenCL
+// RUN: %clangxx -Xclang -fsycl-allow-func-ptr -fno-sycl-device-code-split -std=c++14 -fsycl %s -o %t.out -lOpenCL
```

Comment:
Why exactly is this needed for func-ptr tests? I only have a wild guess, so I'd appreciate an enlightenment.

Reply:
Because now `sycl-post-link` (splitter) doesn't care about indirectly callable functions and just wipes out them from the module (this is my bad, actually). I had offline discussion with @AlexeySachkov , we agreed that split can be disabled for tests with function pointers for now.